services/gossip/topic_block_sync.go

```diff
@@ -52,6 +52,10 @@ func (s *service) receivedBlockSyncAvailabilityRequest(ctx context.Context, head
 	}
 	batchRange := gossipmessages.BlockSyncRangeReader(payloads[0])
 	senderSignature := gossipmessages.SenderSignatureReader(payloads[1])
+	// attempting with talkol to fix issue #437
```

Comment:
you should log values later if they are nil, in case this attempt did not work.. as the underlying issue hardly manifests

Reply:
I'll add a PR to gossip to fix this